common-arcana.lic

```diff
@@ -317,6 +317,7 @@ module DRCA
     return unless data # update_astral_data returns nil on failure
 
     release_cyclics if data['cyclic']
+    fput('release spell') unless checkprep == 'None'
```

Comment:
Would you consider changing the `fput` to a `bput`?

See 0aa0651bf9b9cdad3c170e4cfd5dc4950651b6d0 for an example

Reply:
Done